### PR TITLE
[DNM] Deathsquad buffing

### PR DESCRIPTION
**Pull request by k-k-karma**
:cl: karma
balance: deathsquad now has a x2 bomb cap bomb implant
balance: deathsquad now has martial arts
balance: deathsquad's shield now reflects more (e.g. ballistics, melee, etc)
balance: deathsquad now have no drops and cns rebooter
balance: deathsquad's hardsuit has no slowdown and 90% resist on everything
balance: deathsquad's shield only works if you are a deathsquaddie
/:cl:

[why]: i think ds is a bit underpowered

### CODE REVIEW

**Comment by Lxtoofonea** [comment]
What are the changes?

**Comment by DaydreamPuppeh** [comment]
Seems to buff the shield and give them a martial arts similar to armstrong

**Comment by Lxtoofonea** [comment]
Sure then +1

File: hippiestation/code/datums/martial/elite_cqc.dm [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#define KNOCKOUT_COMBO "HHG"
```
Comment (Manly-Dwarf) [inline hippiestation/code/datums/martial/elite_cqc.dm:1]: Not sure about DS having a better martial art if you ask me.

File: hippiestation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -58,4 +58,11 @@
 
 /obj/item/clothing/suit/space/hardsuit/syndi/elite
 	icon = 'hippiestation/icons/obj/clothing/suits.dmi'
-	alternate_worn_icon = 'hippiestation/icons/mob/suit.dmi'
\ No newline at end of file
+	alternate_worn_icon = 'hippiestation/icons/mob/suit.dmi'
+
+/obj/item/clothing/suit/space/hardsuit/deathsquad
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/clothing/spacesuits/hardsuit.dm:66]: Absurd.

File: hippiestation/code/modules/clothing/outfits/standard.dm [hunk 3]
```diff
@@ -4,4 +4,24 @@
...
 /datum/outfit/wizard/apprentice
 	l_hand = /obj/item/gun/magic/staff
-	backpack_contents = list(/obj/item/storage/box/survival=1)
\ No newline at end of file
+	backpack_contents = list(/obj/item/storage/box/survival=1)
+
+/datum/outfit/death_commando
+	r_pocket = /obj/item/shield/energy/deathsquad
+
+/datum/outfit/death_commando/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/clothing/outfits/standard.dm:13]: ABSURD

**Comment by DaydreamPuppeh** [comment]
Make them look more like squids

**Comment by DaydreamPuppeh** [comment]
Or rename the martial arts to squid style. Because Death squid

File: hippiestation/code/game/objects/items/shields.dm [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    		final_block_chance += 30
+    	if(attack_type == LEAP_ATTACK)
+    		final_block_chance = 100
+            return 1
+        if(attack_type == MELEE_ATTACK)
+           final_block_chance = 90
+       if(attack_type == PROJECTILE_ATTACK)
+           final_block_chance = 70
+           if(istype(hitby, /obj/item/projectile/bullet/neurotoxin))
+               final_block_chance = 100
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/objects/items/shields.dm]: hahahahahah

**Comment by uBadush** [comment]
-1

**Comment by DaydreamPuppeh** [comment]
+1

File: hippiestation/code/modules/clothing/outfits/standard.dm [hunk 5]
```diff
@@ -4,4 +4,24 @@
...
+	cns_imp.stun_cap_amount = 20
+	cns_imp.Insert(H)
+	var/obj/item/implant/explosive/macro/macrobomb = new/obj/item/implant/explosive/macro(H) //you're not killing one and getting away with it
+	macrobomb.heavy = 10
+	macrobomb.medium = 20
+	macrobomb.weak = 30
+	macrobomb.implant(H, null, 1)
+	var/datum/martial_art/elite_cqc/cqc = new/datum/martial_art/elite_cqc()
+	cqc.teach(H)
+	to_chat(H, "<span class='boldannounce'>You know Elite CQC. Make sure you use it.</span>")
```
Comment (yoyobatty) [inline hippiestation/code/modules/clothing/outfits/standard.dm:23]: You do realize outfits have support for implants, you didn't need all this junk.

Comment (k-k-karma) [reply hippiestation/code/modules/clothing/outfits/standard.dm]: This is how the DS outfit does it

**Comment by yoyobatty** [comment]
Please update all changes in the chanelogs and have other headmins re-approve this considering toof gave you a +1 one for thinking it only buffs shields and gives cqc which isn't exactly the case.

**Comment by Lxtoofonea** [comment]
+1

**Comment by yobob591** [comment]
if it works +1

**Comment by JohnMcLaime** [comment]
All this does is change how you fight deathsquad or just avoid them which is easy anyway, making for a less fun (for everyone, except people who are super butthurt about that antag) showdowns.
Might make them flatout invulnerable or add infinite and indestructable ds spawner in their shuttle, cause unrobust squaddies will still get rekt even with this change.

**Comment by HideAndSeekLOGIC** [comment]
I'd +1 something like this if it wasn't so excessive. So far, -1.

**Comment by Carbonhell** [comment]
This has enough +1s. Is the pr ready?

**Comment by HideAndSeekLOGIC** [comment]
@Carbonhell https://www.youtube.com/watch?v=LM8O9WzHDvE

**Comment by Carbonhell** [comment]
DS is only used in extreme cases, so it's okay for it to be ridicolously OP.

**Pull request merged**